src/brain/gocdSlackFeeds/index.ts

```diff
@@ -12,6 +12,7 @@ import { GoCDResponse } from '@/types';
 import { DeployFeed } from './deployFeed';
 
 const ENGINEERING_PIPELINE_FILTER = [
+  'deploy-getsentry-backend-s4s',
```

Comment:
Should this be a constant?

Reply:
No it doesn't add any value - the constants are largely there to make it is easy to run eng-pipes in a test environment (i.e. diff gocd, diff github org, diff repos)

Reply:
fair enough